README.md

```diff
@@ -23,60 +23,77 @@ Additional functionality may be built out in the future.
     export REQUEST_SIGNER_COGNITO_PASSWORD=''
   ```
 
-## User Quickstart
+## Installation
 
-1. Clone the repository
+### Unix and Unix-like Systems
+1. Create a virtual environment
     ```
-   git clone https://github.com/NASA-PDS/registry-client.git
-   cd registry-client
-    ```
-   
-
-2. Create a virtual environment
-    ```
-    python -m venv venv
-    source ./venv/bin/activate
+    # Example assumes bash command shell. For others, consult shell documentation.
+    mkdir -p $HOME/.virtualenvs
+    python3 -m venv $HOME/.virtualenvs/registry-client
+    source $HOME/.virtualenvs/registry-client/bin/activate
     ```
 
-3. Install the tool to the virtual environment
+2. Install the tool to the virtual environment
     ```
-    pip install .
+    pip install pds.registry-client
     ```
 
-4. Run the tool directly
+3. Run the tool directly
     ```
-    ./cli/makesignedrequest --help
+    registry-client --help
     ```
+### Windows Installation
+For help installing Python, [see this documentation from PDS Deep Archive](https://nasa-pds.github.io/deep-archive/installation/index.html#installing-python-for-windows).
 
----
+To install the Registry Client software on Windows comprises the following steps:
 
-## BEGIN BOILERPLATE
+Installing Python 3.11 for Windows
+Creating a “virtual environment” to contain an isolated instance of Python 3.11
+Installing LXML 4.9.0 for Python 3.11 into the virtual environment
+Installing the PDS Deep Archive into the virtual environment
```

Comment:
Is the Deep Archive required in order to run the Registry Client?

Reply:
--no-verify